test/test.js

```diff
@@ -34,17 +45,18 @@ describe('cacheOnDemand', function(){
       });
     }
   });
-  it('does the work only once for those ten invocations', function() {
+  it('does the work only once for those ten invocations', () => {
```

Comment:
Ok interesting, saw a lot of tests with arrow functions in A3.

Reply:
I take your point that tests with arrow functions have already crept into A3, and I'm not going to ask you to rewrite all of these, but in future I'd like us to avoid it because it just creates a problem for those using standard mocha functionality like `this.setTimeout()`. Which I realize isn't needed in this case.